core/src/main/java/com/linecorp/armeria/common/util/BlockingTaskExecutorBuilder.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2021 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common.util;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static java.util.Objects.requireNonNull;
+
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+
+import com.linecorp.armeria.common.CommonPools;
+import com.linecorp.armeria.common.Flags;
+
+/**
+ * Provides a builder that is useful for creating an BlockingTaskExecutor.
+ *
+ * @see CommonPools#blockingTaskExecutor()
+ */
+public final class BlockingTaskExecutorBuilder {
+
+    private static final String DEFAULT_THREAD_NAME_PREFIX = "armeria-common-blocking-tasks";
+    private static final int DEFAULT_KEEP_ALIVE_TIME_SECONDS = 60;
+
+    private String threadNamePrefix;
+    private int numThreads;
+    private long keepAliveTime;
+    private boolean allowCoreThreadTimeout;
+
+    private BlockingTaskExecutorBuilder() {
+        threadNamePrefix = DEFAULT_THREAD_NAME_PREFIX;
+        numThreads = Flags.numCommonBlockingTaskThreads();
+        keepAliveTime = DEFAULT_KEEP_ALIVE_TIME_SECONDS;
+        allowCoreThreadTimeout = true;
```

Comment:
We usually assign these default values at the field definitions. Could you do that, please?